homeassistant/components/system_bridge/sensor.py

```diff
@@ -129,6 +146,14 @@ def memory_used(data: SystemBridgeCoordinatorData) -> float | None:
             data.system.boot_time, tz=timezone.utc
         ),
     ),
+    SystemBridgeSensorEntityDescription(
+        key="cpu_power_package",
+        name="CPU Package Power",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        icon="mdi:chip",
+        value=cpu_power_package,
+    ),
```

Comment:
Add `suggested_display_precision=2` instead of rounding it in `cpu_power_per_cpu`.